pkg/capture/crd_to_job_test.go

```diff
@@ -135,6 +138,9 @@ func Test_CaptureTargetsOnNode_AddNodeInterface(t *testing.T) {
 }
 
 func Test_CaptureToPodTranslator_GetCaptureTargetsOnNode(t *testing.T) {
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGTERM)
```

Comment:
In tests, you can do this slightly better with a `context.WithDeadline`. You can get the deadline with `testing.(*T).Deadline()` which is bound to the `-timeout` value of `go test`. I usually wrap that all up in a `testContext` helper or something that I just pass the `testing.(*T)` to.